package.json

```diff
@@ -59,6 +59,8 @@
     "test-limiter": "npm run test-with-coverage -- tests/limiter/limiter.js",
     "test-main": "npm run test-with-coverage -- --recursive tests/api tests/clock tests/config tests/db tests/enforceToken tests/realtime tests/enableCache tests/logging",
     "test-travis": "npm run test && npm run coveralls",
+    "test-routes-enabled": "HIDE_ROUTES=true HIDDEN_ROUTES=aspects mocha -R dot --recursive tests/routeFlag/FlagEnabled",
```

Comment:
@BrandonMurnane these test commands aren't getting called as part of the travis build.. I think you can add the calls for these to the end of test-extra or something similar

Reply:
thanks i missed that